src/hooks/use-fetch-wpcom-sites/index.tsx

```diff
@@ -51,6 +52,7 @@ export const sitesEndpointResponseSchema = z.object( {
 } );
 
 const STUDIO_SYNC_FEATURE_NAME = 'studio-sync';
+const A8C_SITE_OWNER_ID = 26957695;
```

Comment:
I don't think exposing this ID is a good approach.
We also do it in calypso, but I believe it would be safer if we could handle this in the API instead.

Reply:
I was thinking about the same as well, but since I have seen the ID used in public Calypso repo already, I followed the same approach. Let's continue the discussion in STU-390-linear-issue.